Jenkinsfile

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/groovy
+def workspace = "/mnt/ssd/jenkins/workspace/SoftwareContainerPipeline"
```

Comment:
Can we really have this hard-coded? We should be able to capture the workspace of the first workspace (build stage) and then use that in runInVagrant. I don't know about groovy scripts, but in Jenkins shell scripts you can use ${WORKSPACE}